### PR TITLE
Feature/stes 601

### PR DESCRIPTION
**Pull request by ayeletlevron**

### CODE REVIEW

File: scm/src/main/java/com/sandboni/core/scm/scope/analysis/VersionScanner.java [hunk 1]
```diff
@@ -15,7 +15,7 @@ public boolean scan(Change change) {
             return false;
         }
 
-        String[] rows = change.getFileContent().orElse("").split(System.lineSeparator());
+        String[] rows = change.getFileContent().orElse("").split("\\R"); //covers Windows/Unix/Mac OS
```
Comment (victor-bonilla) [inline scm/src/main/java/com/sandboni/core/scm/scope/analysis/VersionScanner.java]: If the fileContent is empty, we will still have IndexOutOfBoundsException, right? better to check that scenario too.

File: scm/src/test/java/com/sandboni/core/scm/scope/analysis/VersionScanTest.java [hunk 2]
```diff
@@ -30,6 +31,24 @@
             "    <name>Test SCM</name>\r\n" +
             "</project>";
 
+    private String unixChildPom = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
```
Comment (victor-bonilla) [inline scm/src/test/java/com/sandboni/core/scm/scope/analysis/VersionScanTest.java]: Add a test to empty content

File: scm/src/main/java/com/sandboni/core/scm/scope/analysis/VersionScanner.java (hunk 1 above)
Comment (ayeletlevron) [reply scm/src/main/java/com/sandboni/core/scm/scope/analysis/VersionScanner.java]: Not really. we check change.getFileContent().isPresent() a few lines before (and return false)

File: scm/src/test/java/com/sandboni/core/scm/scope/analysis/VersionScanTest.java (hunk 2 above)
Comment (ayeletlevron) [reply scm/src/test/java/com/sandboni/core/scm/scope/analysis/VersionScanTest.java]: ok
